pybliometrics/utils/get_content.py

```diff
@@ -99,19 +112,44 @@ def get_content(url, api, params=None, **kwds):
         except (IndexError, ValueError):
             pass
 
-    # Perform request, eventually replacing the current key
-    timeout = config.getint("Requests", "Timeout", fallback=20)
-    resp = session.get(url, headers=header, proxies=proxies, params=params,
-                       timeout=timeout)
-    while resp.status_code == 429:
+    # Either use token header or key header
+    if insttoken:
+        header = {**base_header,
+                  'X-ELS-APIKey': token_key,
+                  'X-ELS-Insttoken': insttoken}
+        resp = session.get(url, headers=header, params=params, timeout=timeout)
+    else:
+        header = {**base_header,
+                  'X-ELS-APIKey': key}
+        resp = session.get(url, headers=header, params=params, timeout=timeout, proxies=proxies)
```

Comment:
So proxies may not be used when InstTokens are in place. Can't we just enforce an empty dict and use a common `resp = ...` line of code?

Reply:
The problem is that if we get a 429 or 401 we have to set the proxies again to the ones of the config. I think it is more transparent if we have two `session.get(...)`.